electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -40,7 +43,49 @@ def to_list(self) -> List[Dict[str, Any]]:
         return [event.to_dict() for event in self.events]
 
 
-class ExchangeList(EventList):
+class MergeableEventList(EventList, ABC):
+    """A wrapper around Events lists that can be merged."""
+
+    @classmethod
+    def is_completely_empty(
+        cls, ungrouped_events: Sequence["MergeableEventList"], logger: Logger
+    ) -> bool:
+        """Merge multiple lists of events into one."""
```

Comment:
Can we update this to be more descriptive of just this function, maybe something like:

```suggestion
        """Checks if the lists to be merged has any data."""
```

What do you think?

Reply:
Yes I have added it now.